streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBKeyValueStoreTest.java

```diff
@@ -86,19 +87,21 @@ public void setConfig(final String storeName, final Options options, final Map<S
         }
     }
 
+    @After
+    public void after() {
+        context.close();
+    }
+
     @Test
     public void shouldUseCustomRocksDbConfigSetter() throws Exception {
-        final KeyValueStoreTestDriver<Integer, String> driver = KeyValueStoreTestDriver.create(Integer.class, String.class);
         driver.setConfig(StreamsConfig.ROCKSDB_CONFIG_SETTER_CLASS_CONFIG, TheRocksDbConfigSetter.class);
         createKeyValueStore(driver.context(), Integer.class, String.class, false);
         assertTrue(TheRocksDbConfigSetter.called);
     }
 
     @Test
     public void shouldPerformRangeQueriesWithCachingDisabled() throws Exception {
-        final KeyValueStoreTestDriver<Integer, String> driver = KeyValueStoreTestDriver.create(Integer.class, String.class);
-        final MockProcessorContext context = (MockProcessorContext) driver.context();
-        final KeyValueStore<Integer, String> store = createStore(context, Integer.class, String.class, false, false);
+        final KeyValueStore<Integer, String> store = createStore(context, Integer.class, String.class, false);
```

Comment:
We can make it a class member, but we cannot initialize globally because `shouldUseCustomRocksDbConfigSetter` uses a different `driver` setting to create the store.